.github/workflows/linux.yml

```diff
@@ -24,146 +24,101 @@ jobs:
-            os: ubuntu-20.04
+            os: ubuntu-22.04
           - CC: clang-13
             CXX: clang++-13
-            PackageDeps: clang-13
-            Repo: llvm-toolchain-$(lsb_release -cs)-13
             platform_file: include $(ACE_ROOT)/include/makeinclude/platform_linux_clang.GNU
-            os: ubuntu-20.04
+            os: ubuntu-22.04
           - CC: clang-14
             CXX: clang++-14
-            PackageDeps: clang-14
-            Repo: llvm-toolchain-$(lsb_release -cs)-14
             platform_file: include $(ACE_ROOT)/include/makeinclude/platform_linux_clang.GNU
-            os: ubuntu-22.04
+            os: ubuntu-24.04
+            PackageDeps: clang-14
           - CC: clang-15
             CXX: clang++-15
-            PackageDeps: clang-15
-            Repo: llvm-toolchain-$(lsb_release -cs)-15
             platform_file: include $(ACE_ROOT)/include/makeinclude/platform_linux_clang.GNU
-            os: ubuntu-22.04
+            os: ubuntu-24.04
+            PackageDeps: clang-15
           - CC: clang-16
             CXX: clang++-16
-            PackageDeps: clang-16
-            Repo: llvm-toolchain-$(lsb_release -cs)-16
             platform_file: include $(ACE_ROOT)/include/makeinclude/platform_linux_clang.GNU
-            os: ubuntu-22.04
+            os: ubuntu-24.04
           - feature: CORBA/e micro
             CC: gcc-10
             CXX: g++-10
             PackageDeps: g++-10
             platform_file: include $(ACE_ROOT)/include/makeinclude/platform_linux.GNU
-            os: ubuntu-20.04
+            os: ubuntu-24.04
             optional_feature: corba_e_micro=1
             optional_macros: corba_e_micro=1
           - feature: CORBA/e compact
             CC: gcc-10
             CXX: g++-10
             PackageDeps: g++-10
             platform_file: include $(ACE_ROOT)/include/makeinclude/platform_linux.GNU
-            os: ubuntu-20.04
+            os: ubuntu-24.04
             optional_feature: corba_e_compact=1
             optional_macros: corba_e_compact=1
           - feature: ACE for TAO
             CC: gcc-10
             CXX: g++-10
             PackageDeps: g++-10
             platform_file: include $(ACE_ROOT)/include/makeinclude/platform_linux.GNU
-            os: ubuntu-20.04
+            os: ubuntu-24.04
             optional_feature: ace_for_tao=1
             optional_macros: ace_for_tao=1
           - feature: wchar
             CC: gcc-10
             CXX: g++-10
             PackageDeps: g++-10
             platform_file: include $(ACE_ROOT)/include/makeinclude/platform_linux.GNU
-            os: ubuntu-20.04
+            os: ubuntu-24.04
             optional_feature: uses_wchar=1
             optional_macros: inline=0
           - feature: CodeQL
             CC: gcc-10
             CXX: g++-10
             PackageDeps: g++-10
             platform_file: include $(ACE_ROOT)/include/makeinclude/platform_linux.GNU
-            os: ubuntu-20.04
+            os: ubuntu-24.04
```

Comment:
@jwillemsen looks like the CodeQL configuration in GitHub needs to be changed so it can work with 24.04 instead of 20.04

Reply:
As far as I can find there is nothing to configure on github

Reply:
OK, I guess it resolved itself, but when I was working on this I saw an error on https://github.com/DOCGroup/ACE_TAO/security/code-scanning/tools/CodeQL/status